lib/rucio/core/naming_convention.py

```diff
@@ -139,8 +138,12 @@ def validate_name(scope, name, did_type, session=None):
     groups = match(regexp, str(name))
     if groups:
         meta = groups.groupdict()
-        if 'task_id' in meta and meta['task_id']:
-            meta['task_id'] = int(meta['task_id'])
+        # Hack to get task_id from version
+        if 'version' in meta and meta['version']:
+            matched = match('(?P<version>\w+)_tid(?P<task_id>\d+)_\w+$', meta['version'])
+            if matched:
+                meta['version'] = matched.groupdict()['version']
```

Comment:
I think we need a check that length(version) > 50 there (column limitation)

Reply:
We don't do it for the other columns. If we try to insert a version longer than 50, the core will just throw this exception : Details: [u'(cx_Oracle.DatabaseError) ORA-12899: value too large for column "ATLAS_RUCIO"."DIDS"."VERSION" (actual: 57, maximum: 50)\n']Traceback (most recent call last): Details: [u'(cx_Oracle.DatabaseError) ORA-12899: value too large for column "ATLAS_RUCIO"."DIDS"."VERSION" (actual: 57, maximum: 50)\n']